testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveStorageFormats.java

```diff
@@ -660,6 +661,42 @@ else if (!(isParquetStorageFormat && isParquetOptimizedWriterEnabled)) {
         onTrino().executeQuery(format("DROP TABLE %s", tableName));
     }
 
+    @Test(groups = STORAGE_FORMATS_DETAILED)
+    public void testTimestampFieldWrittenByOptimizedParquetWriterCannotBeReadByHive()
+    {
+        // only admin user is allowed to change session properties
+        setAdminRole();
+        setSessionProperties(onTrino().getConnection(), Map.of("hive.experimental_parquet_optimized_writer_enabled", "true"));
+
+        String tableName = "parquet_table_timestamp_created_in_trino";
+
+        onTrino().executeQuery("DROP TABLE IF EXISTS " + tableName);
+
+        onTrino().executeQuery("CREATE TABLE " + tableName + " WITH (format='PARQUET') AS SELECT TIMESTAMP '2015-05-10 12:15:35.123' as ts");
```

Comment:
this can be more terse

```suggestion
        String tableName = "parquet_table_timestamp_created_in_trino";
        onTrino().executeQuery("DROP TABLE IF EXISTS " + tableName);
        onTrino().executeQuery("CREATE TABLE " + tableName + " WITH (format='PARQUET') AS SELECT TIMESTAMP '2015-05-10 12:15:35.123' as ts");
```

Reply:
👍